component/badge/badge.css

```diff
@@ -39,6 +40,21 @@
   white-space: nowrap;
 }
 
+.badge-round-sm {
+  height: 8px;
+  width: 8px;
+}
+
+.badge-round-md {
+  height: 1rem;
+  width: 1rem;
+}
+
+.badge-round-lg {
+  height: 1.12rem;
+  width: 1.12rem;
+}
+
 .badge-sm {
   font-size: 12px;
```

Comment:
Use rem unless you have a purpose for px.

Reply:
i am using rem if the value is greater or equal to 16px